packages/salesforcedx-vscode-replay-debugger/src/adapter/apexReplayDebug.ts

```diff
@@ -101,9 +148,7 @@ export class ApexReplayDebug extends DebugSession {
 
   public threadsRequest(response: DebugProtocol.ThreadsResponse): void {
     response.body = {
-      threads: [
-        new Thread(ApexReplayDebug.THREAD_ID, this.logContext.getLogFileName())
-      ]
+      threads: [new Thread(ApexReplayDebug.THREAD_ID, '')]
```

Comment:
Thread name is only useful if the debugger supports multiple debugged threads. If there's only one, VS Code doesn't show the name.